### PR TITLE
Update README section on ArchivesSpace data

### LINKED ISSUE #359: Document how staging and production ASpace should be used

### PR DESCRIPTION
**Pull request by marlo-longley**
Fixes #359

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,46 +21,54 @@ We also added a `video.xml` file in fixtures that only contains the records/drui
...
+Making requests to the ArchivesSpace endpoints via rake task requires certain environment variables be set. Locally you can pass them on the command line or set them up another way. On our `vt-stage` and `vt-prod` servers running this application, these env vars are stored and accessible. Both `vt-stage` and `vt-prod` are set in Vault to point to `http://spec-as-prod.stanford.edu` (production ArchivesSpace) as the source for their EAD. The following variables are needed to perform the download:
 
-### Required environment variables
-Making requests to the ArchivesSpace endpoints via rake task requires that certain environment variables are set. Locally you can pass them on the command line or set them up another way. On staging and production servers, they are stored and accessible.
 - `ASPACE_URL`
 - `ASPACE_USER`
 - `ASPACE_PASSWORD`
 
-You can get the values of Vault by running the following (stage example):
+You can get these values frome Vault by running the following (stage example):
```
Comment (thatbudakguy) [inline README.md:35]: tiny typo, "frome"

File: README.md [hunk 2]
```diff
@@ -21,46 +21,54 @@ We also added a `video.xml` file in fixtures that only contains the records/drui
...
+1. Find out the ID of the Resource and its Resository that you want in ArchivesSpace. Repository refers to internal SUL organization. On the production ArchivesSpace server, the Repository ID is `18`, for Virtual Tribunals. "Resource" is the name for top level containers in ArchivesSpace. The Resource ID is the permanent identifier we use for the collection. It's the same as the druid and won't change: `mt839rq8746`
 
-3. Download the EAD for a specific Resource. This will download EAD to our `/data` folder with the Resource ID as the filename, e.g. `/data/mt839rq8746.xml`. You can use the following command. When trying to pass arguments to a rake task in the form of `[arg1,arg2]`, it seems my zsh shell requires escaping the brackets:
+2. Run the download task -- it takes resource and repository as arguments. When trying to pass arguments to a rake task in the form of `[arg1,arg2]`, it seems my zsh shell requires escaping the brackets:
     ```shell
     rake vt:download_resource\[18,mt839rq8746\]           
     ```
 
     Locally with passing env variables, the command could like like this:
```
Comment (thatbudakguy) [inline README.md:51]: "could like like"

File: README.md [hunk 3]
```diff
@@ -21,46 +21,54 @@ We also added a `video.xml` file in fixtures that only contains the records/drui
...
+
+3. Index the file.
     ```shell
      rake vt:index 
     ```
-5. Clear the caches
+4. If indexing on a remote server, we need to clear the caches:
    ```shell
     cap prod remote_execute["cd vt/current; RAILS_ENV=production bin/rails tmp:cache:clear"]
    ```
```
Comment (thatbudakguy) [inline README.md:67]: is it worth adding another step here to mention when to commit the changed data? should we say that any time you pull EAD XML, you should commit it if it's changed?

**Review by thatbudakguy — APPROVED** [review]
thanks for adding this! down to merge it, just added some small thoughts

File: README.md [hunk 4]
```diff
@@ -21,46 +21,54 @@ We also added a `video.xml` file in fixtures that only contains the records/drui
...
 
+### Note on the data in ArchivesSpace
+The data we upload to ArchivesSpace is generated from a custom pipeline. See [/source_data/README.md]().
+Edits can be made to individual records manually, or on a larger scale by rerunning the steps in our data pipeline, and reuploading files to ArchivesSpace. When chanages are made to the collection in production ArchiveSpace, you should download the new EAD, and reindex on  `vt-stage` and `vt-prod`, as described above.
```
Comment (marlo-longley) [inline README.md:70]: @thatbudakguy I think section addresses your comment above, but I'm happy to move it if it makes more sense.

Comment (marlo-longley) [reply README.md]: @thatbudakguy good call. Lmk how that looks!

File: README.md [hunk 5]
```diff
@@ -21,46 +21,54 @@ We also added a `video.xml` file in fixtures that only contains the records/drui
...
     ```
-5. Clear the caches
+4. If indexing on a remote server, we need to clear the caches:
    ```shell
     cap prod remote_execute["cd vt/current; RAILS_ENV=production bin/rails tmp:cache:clear"]
    ```
 
+### Note on the data in ArchivesSpace
+The data we upload to ArchivesSpace is generated from a custom pipeline. See [/source_data/README.md]().
+Edits can be made to individual records manually, or on a larger scale by rerunning the steps in our data pipeline, and reuploading files to ArchivesSpace. When chanages are made to the collection in production ArchiveSpace, you should download the new file as described aboce, and save it in `public/data`, overwriting the existing file. Commit this new file and submit a PR. Once merged and deployed, you can reindex on  `vt-stage` and `vt-prod`, as described above.
```
Comment (thatbudakguy) [inline README.md:70]: this is perfect, thanks!

**Pull request merged**